src/components/input-with-icon/InputWithIcon.styles.ts

```diff
@@ -4,7 +4,8 @@ export const styles = {
     alignItems: 'center',
     px: { xs: '8px', sm: '14px' },
     py: { xs: '7px', sm: 0 },
-    gap: '12px'
+    gap: '12px',
+    minHeight: { xs: '40px', sm: 0 }
```

Comment:
do according to Figma. with your approach it will be bigger than it should be

Reply:
done